### ISSUE TITLE
Add "Turn 180"

### ISSUE THREAD

**Issue by tyras2** [cyberjunk/meridian59-dotnet#95]
For myself i play ogre "the old fashion way" without using the mouse. So i miss the possibility to hotkey a 180 turn. Maybe we can add this feature?

**Comment by cyberjunk** [comment]
Yes this is a valid enhancement request, hopefully will find some time to add it.
